### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by Irnes05**
Missing letters have been added and one sentence has been corrected in the set. My in game ID 6893550

### CODE REVIEW

File: res/values-hr/strings.xml [hunk 1]
```diff
@@ -597,8 +597,8 @@
     <string name="PRIZES">"NAGRADE"</string>
 
     <string name="Enable_Custom_Skin">"Omogući Prilagođene Skinove"</string>
-    <string name="Use_Custom_Skin">"Prilagođeni\nskin"</string>
-    <string name="custom_skin_description">"Ova značajka omogućuje vam da koristite svoj profil sliku ili datoteku kao svoj skin. Ova postavka će samo promjeniti skin koji vi vidite. Drugi ljudi neće vidjeti vaš vlastiti skin."</string>
+    <string name="Use_Custom_Skin">"Koristiti Prilagođenu Kožu"</string>
```
Comment (Jeii20) [inline res/values-hr/strings.xml:600]: Please return the `\n` between "Prilagođenu" and "Kozu".

**Review by Jeii20 — CHANGES_REQUESTED** [review]

File: res/values-hr/strings.xml (hunk 1 above)
Comment (Irnes05) [reply res/values-hr/strings.xml]: > Molimo vratite `\n`između "Koristiti" i "Prilagođenu"; "Kožu" je prebukvalno, pa ga zadržite kao "Skin".

Promjene su poslate

**Review by Jeii20 — APPROVED** [review]
Thanks for your time, @Irnes05. ^^
Please mark the change request conversation as resolved.
Approved the changes, can merge into main as soon as possible.

**Comment by Jeii20** [comment]
By the way, you literally undone the translation you made. I just said to redo the \n and the "Skin". Your fix attempt was correct (with issue).

**Comment by Irnes05** [comment]
> Usput, doslovno ste poništili prevod koji ste napravili. Upravo sam rekao da ponovim \n i "Skin". Vaš pokušaj popravljanja je bio ispravan (sa problemom).

Does that mean I have to do the same translation thing again?

**Comment by Jeii20** [comment]
Yes but just keep the \n and the "Skin", since it's properly used in every other language. ^^

**Pull request closed without merging**